typescript/infra/src/infrastructure/monitoring/prometheus.ts

```diff
@@ -78,7 +78,7 @@ async function getPrometheusConfig(
             {
               action: 'keep',
               regex:
-                '(container.*|optics.*|Optics.*|prometheus.*|ethereum.*|hyperlane.*|kube_pod_status_phase|kube_pod_container_status_restarts_total)',
+                '(container.*|optics.*|Optics.*|prometheus.*|ethereum.*|hyperlane.*|kube_pod_status_phase|kube_pod_container_status_restarts_total|kube_pod_container_resource_requests)',
```

Comment:
Where can I find a list of all the available metrics here?

Reply:
I port forwarded to the on-cluster prometheus server to play around, but also https://github.com/kubernetes/kube-state-metrics